boards/renesas/ek_ra6e2/doc/index.rst

```diff
@@ -98,6 +98,8 @@ The below features are currently supported on Zephyr OS for EK-RA6E2 board:
 +-----------+------------+----------------------+
 | CLOCK     | on-chip    | clock control        |
 +-----------+------------+----------------------+
+| ADC       | on-chip    | adc                  |
```

Comment:
Wrong commmit message. This shoud start with `boards: renesas: ...`

Reply:
also with this issue